clayui.com/src/components/Editor/index.js

```diff
@@ -18,6 +19,7 @@ const Editor = ({
 	code,
 	disabled = false,
 	imports,
+	jspCode,
```

Comment:
I was thinking, instead of having a jsp specific prop, what if we made the `code` prop more flexible, meaning it could handle either a single block of code or an array of objects that have `{name, code}`. This would give us flexibility to add as many named tabs as we want. We could end up having

```js
[{
    name: 'React',
    code: '<SomeComponent'  
},
{
    name: 'JSP',
    code: '<clay-taglib />'  
},
{
    name: 'CSS',
    code: '<div class="container></div>  
}]
```